src/pe_source/data/sixgill/config.py

```diff
@@ -0,0 +1,45 @@
+"""Get cybersixgill API token."""
+
+# Standard Python Libraries
+from configparser import ConfigParser
+import glob
+import os
+
+# Third-Party Libraries
+import requests
+
+# Configuration
+SECTION = "sixgill"
+BASE_DIR = os.path.abspath(os.path.join(__file__, "../../../.."))
+REPORT_DB_CONFIG = glob.glob(f"{BASE_DIR}/**/*.ini", recursive=True)[0]
```

Comment:
This is pretty gnarly- can you do something more like [this](https://github.com/cisagov/pe-reports/blob/develop/src/pe_reports/data/config.py#L9)?